### DISCUSSION TITLE
Can we "take inventory" of alternative CMB explanations?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#179] [Cosmology Models]
I realize that many people here (myself included) have their pet theories. 

But perhaps maybe we can compile a list of non-standard ideas about the CMB? 

I think there might be two broad categories to consider. Ideas where the CMB is somehow related to redshifts, and ideas where they are completely unrelated. 

I don't have any great ideas. But two that keep popping up in my mind are:

1. (directly related to redshifts) 

Assuming a cosmological horizon, light from Earth won't reach past the horizon, and light from the horizon won't reach past Earth. What about light that enters our observable region though, but can't get to Earth?

Something like the picture on the far right.

![](https://raw.githubusercontent.com/mikehelland/hubbles-law/master/img/horizon.png)

It seems the energy that enters our observable region, and then redshifts to oblivion might somehow still be detectable. The energy coming in "dieing" here should raise the temperature I would think.

2. (not directly related to redshifts

We know that hydrogen clouds absorb photons, creating the Lyman alpha forest and Lyman alpha break.

What happens to those photons? What happens to that energy? It seems the atom that absorbed it should re-emit it at some point, in some random direction. Could it statistically work out that the CMB is just static from all the hydrogen out there absorbing extragalactic light?

Anyways. Maybe we can make a "master list" of sorts. Let's not worry so much about debating them or finding in flaws in them. Just on creating a record of them first. Sound reasonable?

**Comment by RedshiftDrift** [comment]
A third idea to add to your list, the CMB is explained by the process [I described earlier](https://github.com/orgs/a-cosmology-group/discussions/58#discussioncomment-6310746).  Briefly:

Redshift is produced by a tired-light process in electrons, which removes small amounts of energy from travelling photons.  (No space expansion is assumed, no horizon is assumed, a Euclidean metric is used.)  The energy removed from redshifted electromagnetic radiation heats the intergalactic electrons to high temperatures $(>10^{10} \ \mbox{K})$.

Those extremely hot electrons cause two effects on the CMB:
1) the CMB is redshifted by the tired-light process,
2) the CMB is blue-shifted by the [Sunayev-Zeldovich process](https://en.wikipedia.org/wiki/Sunyaev%E2%80%93Zeldovich_effect).

In our part of the universe the equilibrium between the redshift and blue-shift processes become equal in magnitude at a temperature of 2.7 K.

**Comment by ExpEarth** [comment]
There is a very scholarly article on this topic by Cirkovic and Perovic which covers the history of CMB interpretations and some of the non-Big Bang ones.  Some of those might be added to the list.  They mention Hoyle, Arp, Eric Lerner and others.  Unfortunately, it's hard to easily itemize them for list purposes.
[CMB_Cirkovic.pdf](https://github.com/a-cosmology-group/acg/files/13230885/CMB_Cirkovic.pdf)

Then there is of course my own mechanism, optical gravity.  I argue that the energy lost in the redshift of the CMB goes into gravity.  In my original paper, attached below, I thought the local CMB could do it.  I later realized it has to be the remote CMB, which as mentioned in another thread, has much higher temperatures.
[GravityfromRefractionFinal.pdf](https://github.com/a-cosmology-group/acg/files/13230898/GravityfromRefractionFinal.pdf)

**Reply by HanDeBruijn** [reply]
>Then there is of course my own mechanism [ .. ]

Let's see. Shortly after equation (20) in the [paper](https://github.com/a-cosmology-group/acg/files/13230898/GravityfromRefractionFinal.pdf) you write:

![nope](https://github.com/a-cosmology-group/acg/assets/135433222/768da0b8-fb25-447f-aec3-3addaca5b957)

**Calculating the gravitational constant** $G$ ?? That would be so revolutionary that I don't believe it beforehand.
But okay, let's give it a chance. On top of page 404 you write:

$$
4\pi G\rho/H_0 \cong H_0 \quad \mbox{or} \quad H_0^2 = 4\pi G\rho \quad \mbox{(?)}
$$

Sort of a _Hubble tension_ I suppose. &Lambda;CDM says that $H_0 = \sqrt{8\pi/3.G\rho_0}$ . And there are similar other expressions.
Whatever. Eq. (16) on the same page reads:

$$
h = \frac{H_0}{\rho c}
$$

So, according to eq. (20), with the energy density $u$ of the CMB plugged in from experiment (i.e. not from theory):

$$
G = \frac{uh^2}{3} = \frac{u H_0^2}{3 \rho^2 c^2} = \frac{u . 4\pi G\rho}{3 \rho^2 c^2} = G \frac{4\pi.u}{3\rho c^2}
$$

It follows that ${\bf G \neq G}$ unless $u = 3/(4\pi) \rho c^2$ . And there are no independent theoretical expressions for $u$ and $\rho$ .

I'm affraid that you are running in circles. But I shall give anyone the advantage of the doubt.

Maybe your theory is not so far off. Let's compare it with mine, with the &Lambda;CDM opinion for $H_0$.
(Note. My theory gives an accurate value for the Cosmic Microwave Background &Theta;emperature)

$$
\frac{4\sigma}{c}\Theta^4 = \frac{W.\mbox{VMT}.e^{-W}}{32\pi.w}\frac{(H_0 c)^2}{G}
$$

And so (assuming consistency $G=G$) similar to your formula $u = 3/(4\pi) \rho c^2$ :

$$
u = \frac{W.\mbox{VMT}.e^{-W}}{32\pi.w}\frac{8\pi}{3}\rho c^2 = \frac{W.\mbox{VMT}.e^{-W}}{12w}\rho c^2
$$

According to your theory, the constant preceding $\rho c^2$ should be nearly equal to $3/(4\pi)$ .

```
W := 3.383634283;
c := 299792458;
VMT := int(exp(-x)/x^2,x=W..infinity);
wW := 2-exp(-W)*(W^2+2*W+2);
same := W^2*VMT*exp(-W)/(12*wW);
same := evalf(3/(4*Pi));

                      same := 0.00004858092707
                      same := 0.2387324146
```
Alas, I have not been able to improve on this result.

**Update**. For comparison we calculate the CMB temperature as it would be according to @ExpEarth .
```
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H_0 := 73.4; # Riess
H_0 := H_0*1000/Mpc;
rho := H_0^2/(4*Pi*G);
c := 299792458;
u := 3/(4*Pi)*rho*c^2;
sigma := 5.670374419*10^(-8);
Theta := evalf((c/(4*sigma)*u)^(1/4));

                         Theta := 20.91451907
```
Same order of magnitude is found in your thread [Can the CMB temperature be less than 2.7 K ?](https://github.com/orgs/a-cosmology-group/discussions/127), as displayed here again:

![oud](https://user-images.githubusercontent.com/135433222/261391755-861468ac-ff7f-46a6-8d38-8b733bb03117.jpg)

**Comment by HanDeBruijn** [comment]
@ExpEarth writes:
>I argue that the energy lost in the redshift of the CMB goes into gravity.

No Matt, though your writeups have been very inspiring, I find that it is exactly the other way around.
There is a [gravitational self-energy field](http://www.alternatievewiskunde.nl/cosmology/nasa.htm) surrounding the mass of the observable universe.
That self-energy itself is equivalent with a remote mass and therefore **the CMB it emits** is redshifted.
Tired Light models would make the CMB be redshifted along its trip towards the observer, which is **not**.
According to the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) (VMT), it is redshifted **at the rim** of the Observable universe.
So gravity goes into the CMB. And the latter is **uniform** everywhere inside the Observable universe.
[Previous theory](https://github.com/orgs/a-cosmology-group/discussions/135#discussioncomment-6956930) went wrong because, according to the VMT, volumes must be calculated in a non-Euclidean way.
VMT actually replaces GR; so apparent (GR-like) distortion of volumes is less strange than it may seem at first sight.

![both](https://github.com/a-cosmology-group/acg/assets/135433222/87050359-fb29-4f7e-8fcb-974a52702fd1)

We repeat the most important results from that [previous post](https://github.com/orgs/a-cosmology-group/discussions/135#discussioncomment-6956930). A purely mathematical result first:

$$
f(x) = \frac{2-e^{-x}(x^2+2x+2)}{x} \quad \mbox{with} \quad W = 3.383634283 \quad \mbox{and} \quad w = f(W) = 0.3883945571
$$

For the mass $M_O$ and radius $R_O$ of our Observable universe we find (with $c=$ lightspeed, $G=$ gravitational constant, $H=$ Hubble parameter):

$$
M_O = \frac{W}{2}\frac{c^3}{G.H} \quad \mbox{;} \quad R_O = W\frac{c}{H}
$$

The total gravitational self energy of our Observable universe has been correctly calculated so far, but as a Variable Mass (with $E=mc^2$) it must be redshifted at the rim with the maximal observable one, namely $1+z_{max} = e^W$ and so:

$$
U_G = - \frac{G M_O^2}{2R_O}.W.\mbox{VMT} / e^{W} \quad \mbox{where} \quad \mbox{VMT} = \int_W^\infty \frac{e^{-x}}{x^2}dx
$$

Furthermore it went **wrong** (with $\sigma=$ [Stefan-Bolzmann constant](https://en.wikipedia.org/wiki/Stefan%E2%80%93Boltzmann_law), $\Theta=$ temperature) at the place **where**

$$
U_{CMB} = \frac{4}{3}\pi R_O^3.\frac{4\sigma}{c}\Theta^4
$$

When multiplied with mass density instead of radiation density, the mass of the Observable universe can never be greater than the mass of the whole Universe. The latter has been calculated in an [earlier stage](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6826896). We must substitute $\rho_0 \to 4\sigma/c.\Theta^4$ and then formulate correctly instead.

$$
U_{CMB} = \frac{4\sigma}{c}\Theta^4 \int_0^{R_O} e^{-H/c.r}.4\pi r^2.dr =
\frac{4\sigma}{c}\Theta^4 \left(\frac{c}{H}\right)^3 4\pi \left[\frac{2-e^{-W}(W^2+2W+2)}{W}\right]W = 4\pi.wW \left(\frac{c}{H}\right)^3 \frac{4\sigma}{c}\Theta^4
$$

Leading to the following end-result.

$$
\frac{4\sigma}{c}\Theta^4 = \frac{W.\mbox{VMT}.e^{-W}}{32\pi.w}\frac{(Hc)^2}{G}
$$

```
W := 3.383634283;
G := 6.67408*10^(-11);
Mpc := 3.08567758*10^22;
H := 73.4; # Riess
H := H*1000/Mpc;
c := 299792458;
VMT := int(exp(-x)/x^2,x=W..infinity);
sigma := 5.670374419*10^(-8);
wW := 2-exp(-W)*(W^2+2*W+2);
u_G := W^2*VMT*exp(-W)/(32*Pi*wW)*(H*c)^2/G;
Theta := evalf((c/(4*sigma)*u_G)^(1/4));

                         Theta := 2.764453563
```
The small discrepancy with experiment that is remaining can easily be explained by the uncertainty of the Hubble parameter.
Calculating the other way around:
```
Theta := 2.726;
H := sqrt((G/c^2)*(32*Pi*wW)/(W^2*VMT*exp(-W))*4*sigma/c*Theta^4);
H := evalf(H/1000*Mpc);

                           H := 71.37221336
```
Which is compatible with the Hubble tension.
Sorry for the earlier postings. Where wood is chopped, chips fall.

**Reply by ExpEarth** [reply]
@HanDeBruijn 

There is some limited correspondence between our two models.  I actually can't get a good enough handle on yours to really make good comments on it.  And that's even before looking at your equations!

You say that the gravitational self energy of masses emits CMB radiation and this CMB is already redshifted.  I don't really get what the field is that you are talking about, and the link does not help.  Your CMB redshift is sort of like the gravitational redshift I guess.

In my model I would say it differently.  The mutual gravitational potential energy between masses is a real energy form, which undergoes a redshift similar (but not the same) as the Hubble redshift.  I call this the Hubble luminosity, given by _L_= -  _UH_  , where _U_ is the gravitational potential energy.  It is the Hubble luminosity of the entire visible universe which yields the CMB.  This is achieved over one cosmic cycle of period 1/_H_ , since during one such cycle all the CMB photons created will have been reduced to near zero energy.  The energy lost from the CMB redshift goes into gravity, as in the paper I added above.

We should keep in mind that Mike wanted to have a general listing of CMB mechanisms here, not debates about particular ones.

**Reply by HanDeBruijn** [reply]
@ExpEarth 

>We should keep in mind that Mike wanted to have a general listing of CMB mechanisms here, not debates about particular ones.

And in an [earlier post](https://github.com/orgs/a-cosmology-group/discussions/179#discussioncomment-7449138) you write:

>Then there is of course my own mechanism

???

**Reply by HanDeBruijn** [reply]
@ExpEarth .
>I don't really get what the field is that you are talking about

Our observable universe is a spherical body with a mass.
Around that sphere there is a gravitational [Self Energy Field](http://www.alternatievewiskunde.nl/cosmology/nasa.htm),  like the electrical field around a charged body.

With a caveat, though. Our (purported completion of the) **Variable Mass Theory** is present everywhere.
Mathematically it means that many "common" integranls with mass/energy in them must be replaced.
They become more complicated ones, namely with an additional exponential decay factor inside.
Causing that "ton of math" all over the place :-(

>You say that the gravitational self energy of masses emits CMB radiation and this CMB is already redshifted

The self-energy field itself is a Variable Mass again, at maximally observable cosmic distance this time.
Thus it can only be "seen" **at the rim** that separates the observable universe from the eternal/infinite.

>Your CMB redshift is sort of like the gravitational redshift I guess.

All of the Variable Mass/Energy is observed with **intrinsic redshift** where it originates, nothing else.

Hope this helps.

**Comment by ExpEarth** [comment]
@mikehelland 

> It seems the energy that enters our observable region, and then redshifts to oblivion might somehow still be detectable. The energy coming in "dieing" here should raise the temperature I would think.

On your first example, if the observable universe is not a black hole, then any radiation coming inside it from outside the horizon would be balanced by radiation going from the inside to the outside.  So there would be no temperature change in this case.

If the universe is a black hole, then the radiation coming from outside the horizon is not necessarily replaced by radiation going the other way, though there could be offsetting processes akin to the jetting from supermassive black holes.  If flows don't balance, then I think it would slowly heat up inside.  This energy could however lead to expansion of the cosmic black hole radius, if more mass is added also.

> What happens to those photons? What happens to that energy? It seems the atom that absorbed it should re-emit it at some point, in some random direction. Could it statistically work out that the CMB is just static from all the hydrogen out there absorbing extragalactic light?

On your second example, it should be recalled that most of the energy density in the observable universe is in the CMB.  This is true even for the local CMB with T = 2.7 K.  If the CMB temperature increases with distance, as many studies show, then the CMB in a static universe would have at least 10^4 times as much radiation as all other frequencies combined.  Only a small portion of the starlight would be intercepted by clouds.  Most of it is redshifted away.  So there is a general problem of energy flow balance.  In a static model all the flows should be in equilibrium.  The energy being lost from the CMB each second due to the Hubble redshift is far greater than the amount you are adding to it by only partially absorbed starlight.

**Reply by mikehelland** [reply]
>On your first example, if the observable universe is not a black hole, then any radiation coming inside it from outside the horizon would be balanced by radiation going from the inside to the outside. So there would be no temperature change in this case.

Right. I think that would be helpful though. We can calculate the amount of energy that originates in our observable region and leaves, and that should equal the amount of energy that originates outside the observable region and enters.


>On your second example, it should be recalled that most of the energy density in the observable universe is in the CMB. 

Just curious, how old is that estimate? It seems in the era of JWST we will need to revise those types of figures.

Also, if the Lyman alpha break is kind of "deleting" big parts of a galaxy's spectra, then presumably at some distance it will erase the entire spectra, leaving them invisible (unless by some chance that there is a very significant void between us and it) with all off the light's energy scattered in random directions.

**Reply by ExpEarth** [reply]
> Just curious, how old is that estimate? It seems in the era of JWST we will need to revise those types of figures.

Here is a figure from a 2018 paper.  It shows that the peak of radiation in the CMB is at least 10 times greater than the next highest ones, the COB and CIB, which are both much higher than the remaining ones.  The little purple curve was added by Louis for a hypothetical CMB with T = 1 K.  It would be nice if there was a table somewhere that broke it down on a percentage basis.  

And this figure is just for the local values.  If the CMB temperature increases with distance, as studies suggest, then the CMB will eventually swamp all the others.


![Cosmic spectrum Fig8](https://github.com/a-cosmology-group/acg/assets/136624758/ec0b7eae-d0a8-43ee-9e1c-df2a4c6e2855)



> if the Lyman alpha break is kind of "deleting" big parts of a galaxy's spectra, then presumably at some distance it will erase the entire spectra, leaving them invisible (unless by some chance that there is a very significant void between us and it) with all off the light's energy scattered in random directions.

But by the time it reaches other clouds the galaxy's light will already have been redshifted significantly.  The absorbing clouds are not dense enough to accomplish what you want.

**Reply by HanDeBruijn** [reply]
.@ExpEarth

>it should be recalled that most of the energy density in the observable universe is in the CMB.

Does that make sense? Let's calculate with theory as presented, with $M_O=$ mass of the observable universe and CMB energy.

$$
M_O = \frac{c^3}{G.H}\frac{W}{2} \quad ; \quad \mbox{CMB} = \frac{G M_O^2}{2R_O}.W.\mbox{VMT}/e^{W} = \frac{c^5}{G.H}\frac{W^2}{8}W.\mbox{VMT}.e^{-W}
$$

The quotient of the CMB energy and the energy of common mass in the observable universe is

$$
\frac{\mbox{CMB}}{M_Oc^2} = \frac{W^2}{4}\mbox{VMT}.e^{-W}
$$

```
W := 3.383634283;
VMT := int(exp(-x)/x^2,x=W..infinity);
W^2/4*VMT*exp(-W);
```

The outcome is ${\bf \approx 0.2\times 10^{-3}}$ . Most of the energy density in the observable universe is in common matter.

**Update**. The self-energy of an _electric_ field has been tentatively used for calculating the [Electromagnetic Mass](http://www.alternatievewiskunde.nl/MijnBoek/vanBopp.htm) of an electron.
We can prove that something like this does not work for the gravitational field. Charge is not mass in the first place. And the gravitational field has negative mass. If we assume that the mass of the field adds up to / is substracted from the common mass then that common mass shall be corrected as follows.

$$
M_O := \frac{c^3}{G.H}\frac{W}{2} - \frac{c^5}{G.H.c^2}\frac{W^2}{8}W.\mbox{VMT}.e^{-W} = M_O \left(1-\frac{W^2}{4}\mbox{VMT}.e^{-W}\right)
$$

Thus giving rise to an iterative process of the form $M_O := M_O(1-g)$ where $g=0.1915329972\times 10^{-3}$.
It takes quite some iterations, but in the end the mass $M_O$ of the observable universe turns out to be **zero**:

$$
\lim_{n\to\infty}M_O(1-g)^n=0
$$

Which is of course impossible. The mass of the field does not combine with the common mass. There must exist another mechanism for converting the self-energy of gravitation into something. That something is conjectured to be the CMB.

**Note**. Suppose for example that the zero value is approached one percent, then we have for the number of iterations

$$
n = \left|\frac{\ln(0.01)}{\ln(1-g)}\right| \approx 24,042
$$

**Reply by ExpEarth** [reply]
@HanDeBruijn 

Thanks for pointing out my error there and sorry for causing confusion.  I meant to say most of the energy density of electromagnetic radiation is in the CMB.  I have found that if the CMB energy density is calculated based on the assumption that the average CMB temperature in the observable universe is 27 K, then it has about half the so-called critical density.  Now if we assume that in a black hole universe the mass that induces black hole formation is all baryonic, then the CMB energy density is about one half the critical density.

**Comment by bgreenman** [comment]
All of you should have your alternative theories tested and repeated to see which model or combination of models about alternative CMB  model will prove correct

**Comment by mikehelland** [comment]
@ExpEarth 

> Here is a figure from a 2018 paper. It shows that the peak of radiation in the CMB is at least 10 times greater than the next highest ones, the COB and CIB, which are both much higher than the remaining ones. 

I think this is a good example of what I'm talking about. This is how light we're detecting from various sources, correct?

Well, the light we're detecting has energy that's been diminished from its "rest frame" by 1+z.

That means only half the energy from z>1 is getting to us, and only 25% of the energy from all sources z>3.

That suggests to me that a substantial portion of the light sources in the observable region have a substantial portion of their light's energy missing and possibly scattered in random directions.

That doesn't really fit with my particular theory of redshifts. But as a "cosmic accountant", that would seem to be where to look.

**Reply by ExpEarth** [reply]
@mikehelland 

We all certainly need to be good "cosmic accountants"!  Those of us who favour the static model are in an ideal position to be such, since the volume and mass of the observable universe are both relatively constant.

Now the situation you describe depends on how you interpret the cosmological redshift.  If we omit the expansion interpretation, it leaves two main candidates: a tired light mechanism and some sort of gravitational/time dilation mechanism.

Taking the TL mechanism first, the energy lost with the (1 + z) factor is lost through the redshift process itself.  The cosmic redshift is confirmed spectroscopically with a shift in the absorption lines to the red.  The energy that is lost was in the photons that were _not_ fully absorbed or scattered by gas clouds.  Something else must therefore be causing this energy loss.  It could be an absorption effect, for example, but with the absorption being extremely gradual.  In my own model the energy is absorbed by a photonic spacetime and then goes into gravity.

If the redshift is through some time dilation process, then if it is similar to the gravitational redshift there is no energy lost during the photon's journey through space.  This is the mainstream interpretation.  In your mechanism, if energy is also not lost, then there is a problem with respect to your CMB mechanism for temperature.  The distant sources with _z_ > 3 emit photons that are intrinsically weaker from our perspective.  On their journey towards us they cannot therefore have transferred the 'missing' energy to gas clouds.

**Reply by HanDeBruijn** [reply]
@ExpEarth in defence of @mikehelland .

>there is no energy lost during the photon's journey through space. This is the mainstream interpretation.

That's correct. The mainstream interpretation is equivalent in some sense: take "primordial" as "beyond the observable".

>In your mechanism, if energy is also not lost, then there is a problem with respect to your CMB mechanism for temperature.

Not at at all. Gravitational self-energy is converted into CMB radiation, but **only** at the rim of the observable universe.
There is NO temperature gradient across the observable universe as in your model; CMB temperature is the same everywhere.
Honestly, I think there are more problems with **your** _CMB mechanism for temperature_, as I have hinted in above postings.

**Update**. What I find the most troublesome is your temperature distribution. Which seems to be dependent upon place and time of the observer. Suppose that there is another planet earth, at the space-time position as sketched in the picture below. What is the temperature then and there ? Is it 2.7 K or is it 20 K ?
![effe](https://github.com/a-cosmology-group/acg/assets/135433222/9bec36f0-2726-4aa8-95b3-1d5577f6455b)

**Reply by ExpEarth** [reply]
@HanDeBruijn 

> Not at at all. Gravitational self-energy is converted into CMB radiation, but only at the rim of the observable universe.

Is 'gravitational self-energy' just the mutual gravitational potential energy between masses?  If so, I don't see why it would convert to CMB energy just at one distance, the 'rim'.  If matter in your model is evenly distributed in space, and there is no outer boundary to it, then it should be generating energy everywhere, if there is a decay process at work.

>  What I find the most troublesome is your temperature distribution. Which seems to be dependent upon place and time of the observer. Suppose that there is another planet earth, at the space-time position as sketched in the picture below. What is the temperature then and there ? Is it 2.7 K or is it 20 K ?

At that position it is 20 K.  There is a preferred reference frame in my model.  It is established by the matter situated at the Hubble radius.  That's where most of the observable universe's matter is located in my model.

**Reply by bgreenman** [reply]
HanDeBrujin is correct

**Comment by luisenrique-arch** [comment]
I don't know if any of you are familiar with this particular explanation for the CMB, but I came across it recently and personally I find it very interesting. What do you think?

https://arxiv.org/abs/2504.10510

**Reply by RedshiftDrift** [reply]
The article I co-authored with Martín is referenced in this paper, that's good.

**Reply by HanDeBruijn** [reply]
Have a look at formula (3) in the paper - why didn't I see this before.

$$
z+1=\frac{\nu_0}{\nu}=\frac{h\nu_0}{h\nu}=\frac{E_0}{E}=\frac{m_0c^2}{mc^2}=\frac{m_0}{m}
$$

Thus TL cannot be distinguished from a Variable Mass theory for photons !!
Subsequent formulas (4),(5),(6) are also in agreement with both VM and TL.

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn that's a clear connection.

Question: You postulate that mass increases at $$m/m_0=e^{Ht}$$
from [this discussion](https://github.com/orgs/a-cosmology-group/discussions/265#discussioncomment-10749472).

The CMB problem is viewed from the perspective that primordial photons have weakened in their path (time) to reach us.
Is your argument then that local photons have increased in energy (like mass), and distant photons now only appear relatively weaker?

**Reply by HanDeBruijn** [reply]
@Francois-Zinserling .

Sorry, Francois, I've made a mistake. See pictures below. The **origin** (0) for TL in the paper is different from mine.

### Variable Mass:
![VM](https://github.com/user-attachments/assets/f27478f2-e6a7-4ae9-ba36-e4129a5ee240)

### Tired Light:
![TL](https://github.com/user-attachments/assets/7c03522b-c606-43c5-bfae-71db44585b90)
 &nbsp; ^
 &nbsp; ||
 origin (0) in paper with &nbsp; $\color{red}{1/(1+z)=m/m_0}$ &nbsp; for photons. See also the [discussion here](https://github.com/orgs/a-cosmology-group/discussions/292#discussioncomment-11430296).

**Reply by RedshiftDrift** [reply]
> $1+z = m_0/m$  why didn't I see this before. Thus TL cannot be distinguished from a Variable Mass theory for photons !! -  @HanDeBruijn 

Photons don't have mass.  Also, the interpretation of this equation is that the mass energy of distant objects appears to change with redshift $z$, not time!  That's a very big difference.

There are experimental ways to tell the difference between TL and VM (e.g. quasars associated with galaxies) and while Arp and Hoyle-Narlikar's interpretation was in favour of VM, the TL explanation supports constant mass and photons losing energy to electrons of the IGM.

**Reply by RedshiftDrift** [reply]
@luisenrique-arch that article is not very interesting.
- First the author assumes the "gravitational drag" redshift proposed by Zwicky in the early 30s.  But Zwicky published a few years later that this idea does not work.
- Second, the author assumes that galaxies emit blackbody radiation at a temperature $T_0 (1+z)$, but this puts the Milky Way at the centre of all galaxies within 240 Mpc which coincidentally have the right temperature to produce the desired effect.  This coincidence is not possible.
- Third, the $R_{max} = 240$ Mpc region must be opaque to microwaves, but observations show detection of microwave radiation well beyond 240 Mpc.  Therre is no such opaque region.

That's too many flaws against Carlos Navia's model. :-1:

**Reply by luisenrique-arch** [reply]
@RedshiftDrift 

Thanks for sharing your thoughts, I really appreciate it. Unfortunately, as an interested amateur rather than a qualified expert, I sometimes find myself getting fascinated by certain models that later turn out to have serious problems.

It seems to be a recurring pattern. I come across a model that looks promising, get excited about it, bring it here, then you point out why it doesn't actually work, and the cycle starts all over again.

The thing is, there still isn't a tired-light model that successfully explains all the relevant observations at once, and that includes your own model. So I sometimes get carried away because I'd really like to find a truly complete alternative model, since one doesn't seem to exist yet.

**Reply by RedshiftDrift** [reply]
@luisenrique-arch  Debunking TL models is a good exercise.
  Since I've done that for over 200 TL models, I'll let you debunk the next one so you get some practice!

  No TL mechanism (and no cosmology) is sufficiently developed to explain all observations.
  Most TL mechanisms (and cosmologies like LCDM) lead to predictions that are incorrect (or make no predictions!)

However, all predictions made by STz have been successful so far, without any modification of (or addition to) the basic mechanism first published at the CCC-2.

**Reply by luisenrique-arch** [reply]
@RedshiftDrift 

I already debunked one tired-light model myself: David Crawford's Curvature Cosmology. In the first post I made on this forum, I mentioned that it was probably the model I found most appealing. But after going back and carefully rereading Crawford's latest paper from 2023, I started noticing some inconsistencies and eventually had to abandon it. Which is unfortunate, because Curvature Cosmology was one of the more comprehensive alternatives out there and genuinely attempted to account for a wide range of observations.

That said, I'm not asking for a model that explains every single observation in existence. What I'd like to see is a model that can successfully explain at least the major ones: the cosmological redshift, the CMB and its key features (the acoustic peaks and anisotropies), quasar distribution, galaxy rotation curves, gravitational lensing in galaxy clusters, the time dilation observed in Type Ia supernovae, and some viable mechanism capable of keeping the universe stable and static over long timescales.

If a model could provide convincing explanations for those phenomena alone, I'd already be very satisfied.

**Comment by asheldrick-research** [comment]
In my theory (ECSM) the CMB is a relic of a phase change in what I describe as a superfluid-like finite response medium. It is a signature of when the first radiation, light etc was first freely able to propagate. The CMB TT power spectrum records the initial onset in peak one and the settling of the medium there on after. I have recovered TT and EE to a good precision and I am currently working on TE, if you're interested please take look at my paper.
https://doi.org/10.5281/zenodo.20003390

**Reply by luisenrique-arch** [reply]
@asheldrick-research 

This is what I found when I opened the link you sent: 

> DOI Not Found
> 
> 10.5281/zenodo.20003390
>
> This DOI cannot be found in the DOI System. Possible reasons are:
>
> * The DOI is incorrect in your source. Search for the item by name, title, or other metadata using a search engine.
> * The DOI was copied incorrectly. Check to see that the string includes all the characters before and after the slash and no sentence punctuation marks.
> * The DOI has not been activated yet. Please try again later, and report the problem if the error continues.

**Reply by asheldrick-research** [reply]
Sorry I have fixed it now and the link should divert to my paper.

**Reply by HanDeBruijn** [reply]
My understanding of your paper's content is rien, nothing, niente, or in plain Dutch: drie keer niks.

**Reply by asheldrick-research** [reply]
It's not easy to navigate.. but what i have attempted to achieve is replicate the the CMB by using physics normally associated with superfluids and condensed matter. This paper has derived the TT power spectrum and the the EE polarised spectrum and gives an explanation for for TE cross examination.

**Reply by HanDeBruijn** [reply]
You don't even explain what TT, EE, TE stand for. How are we supposed to know?

**Reply by asheldrick-research** [reply]
Ah I see.. they are the standard measurements of the CMB profile. 
TT is the angular power spectrum of CMB temperature anisotropies. It measures how temperature fluctuations are correlated across angular scales. It is not literally the temperature gradient. EE is the angular power spectrum of E-mode polarization in the CMB.
TE is the cross-correlation between the temperature anisotropy field and the E-mode polarization field. It shows whether temperature fluctuations and polarization patterns are correlated at each angular scale.

TT = temperature anisotropy power spectrum
EE = E-mode polarization power spectrum
TE = temperature–E-mode cross-correlation spectrum
